project/views.py

```diff
@@ -21,12 +68,10 @@ def populate_projects(request):
     """
     Used to Populate Projects in Local Database. Creates entries based on project names present in AVAILABLE_PROJECTS
     config variable in settings.py
-    :param request:
-    :return:
     """
```

Comment:
why are you deleting the comments (parameter and return type) and this is decreasing the code quality.